lambdas/test-lambda/src/test/java/uk/gov/di/data/lep/MaintTest.java

```diff
@@ -3,20 +3,50 @@
 import com.amazonaws.services.lambda.runtime.Context;
 import org.junit.jupiter.api.Test;
 import org.mockito.Mock;
-import uk.gov.di.data.lep.Main;
+import uk.gov.di.data.lep.classes.GroDeathEventEnrichedData;
+import uk.gov.di.data.lep.enums.GroSex;
+
+import java.time.LocalDate;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
 
 class MainTest {
     @Mock
     private Context context;
 
-    private final Main underTest = new Main();
+    private final GroDeathNotificationMinimisation underTest = new GroDeathNotificationMinimisation();
 
     @Test
     void handleRequestReturnsHello() {
-        var result = underTest.handleRequest("h", context);
 
```

Comment:
could we mock out the config for this, and check that for a couple sets of enrichment fields it does the right thing?

also comments, whitespace, and pls rename the test :)